### ISSUE TITLE
Can't send text with url (parseMode MarkdownV2 doesn't work)

### ISSUE THREAD

**Issue by anorudes** [gram-js/gramjs#332]
Hello. 
Couldn't find a single example of how to send a text with a link?
For example: `[test](http://google.com)`
Can you help?
Also I noticed that the library doesn't support parseMode MarkdownV2.

**Comment by painor** [comment]
Yep, Markdown only supports a very limited set of entities. you should use HTML instead because it's easier. <a href="link">Text</a>
